src/timetracker/components/create-project-page/CreateProjectPage.css

```diff
@@ -0,0 +1,34 @@
+@import "tailwindcss/base";
+@import "tailwindcss/components";
+@import "tailwindcss/utilities";
+
+* {
+  font-family: 'Inter', sans-serif;
+}
+
+html {
+  height: 100%;
+}
+
+.create-project {
+    background-image: linear-gradient(
+        to bottom,
+        rgba(48, 188, 237, 100%),
+        rgba(63, 141, 169, 61.33%),
+        rgba(0, 102, 255, 25%)
+      ),
+      url(../img/background.png);
```

Comment:
Make it so the img goes first, then the gradient. Otherwise the pattern is not visible because it sits behind the gradient.

```css
.create-project {
  background-image: url(../img/background.png),
    linear-gradient(
      to bottom,
      rgba(48, 188, 237, 100%),
      rgba(63, 141, 169, 61.33%),
      rgba(0, 102, 255, 25%)
    );
  background-size: cover;
}
 ```

Reply:
I already have the background styles defined in the App.css, maybe there is a way to just reuse them from there.